packages/trader/src/AppV2/Containers/Trade/trade.tsx

```diff
@@ -30,21 +31,30 @@ const Trade = observer(() => {
     const {
         active_symbols,
         contract_type,
+        currency,
         has_cancellation,
         symbol,
         is_accumulator,
         is_market_closed,
         onMount,
         onChange,
+        onChangeMultiple,
         onUnmount,
+        setV2ParamsInitialValues,
     } = useTraderStore();
-    const { trade_types } = useContractsForCompany();
+    const { trade_types, available_contract_types } = useContractsForCompany();
```

Comment:
In `available_contract_types` I added `default_stake` to `config`. `default_stake` is received from `contracts_for` API for each trade type

Reply:
question does this change need to be done on trade.tsx or could it be done lower down in the dom tree

Reply:
Will move in down in the DOM tree

Reply:
Okay, no, rework one more time: moved resetting stake and TP in trade-store (in order not to create multiple requests). default_stake is extracted from hook in Stake component and there we set in to the store.